src/app/ui/form/CheckboxGroupFormField.tsx

```diff
@@ -83,11 +83,11 @@ const MultiCheckboxFormField: FunctionComponent<MultiCheckboxFormFieldProps> = (
     return <FormFieldContainer hasError={ getIn(errors, name) && getIn(errors, name).length }>
         { label }
 
-        <MultiCheckboxControl
+        { options.length > 1 && <MultiCheckboxControl
```

Comment:
What do we display if `options.length` is `1`?

Reply:
We display a control that allows you select all or none with one click `Select All | None`
Doesn't make sense for a single checkbox.